SRC/index.js

```diff
@@ -0,0 +1,46 @@
+// import _ from 'lodash';
+
+import './style.css';
+
+const theList = document.querySelector('.the-list');
+
+const listItem = [
+  {
+    description: 'do stuff 1',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'do stuff 2',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'do stuff 3',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'do stuff 4',
+    completed: false,
+    index: 3,
+  },
+];
```

Comment:
- [ ]  On loading `index.html`, this list of todos should show up on the page. Please fix it.

Reply:
I don't understand this one please explain to me because on my end it's working ok